docs/src/main/sphinx/functions/string.rst

```diff
@@ -134,6 +134,21 @@ String functions
 
     Removes trailing whitespace from ``string``.
 
+.. function:: soundex(char) -> string
+
+   ``soundex`` returns a character string containing the phonetic representation of ``char``.
+    This function is typically used to evaluate the similarity of two expressions phonetically, 
+    i.e., how the string sounds when spoken.
+
+    Here is an example to illustrate the ``soundex`` function::
+
+        SELECT * FROM nation WHERE SOUNDEX(name)  = SOUNDEX('CHYNA');
```

Comment:
Maybe break into multiple lines and only return name?

```
SELECT name 
FROM nation
WHERE SOUNDEX(name)  = SOUNDEX('CHYNA');
```

Reply:
Done!